package/src/contexts/messageOverlayContext/MessageOverlayContext.tsx

```diff
@@ -47,7 +40,7 @@ export type MessageOverlayData<
   handleReaction?: (reactionType: string) => Promise<void>;
   images?: Attachment<At>[];
   message?: MessageType<At, Ch, Co, Ev, Me, Re, Us>;
-  messageActions?: MessageAction[];
+  messageActions?: MessageActionListItemProps[];
```

Comment:
Should `messageActions` be named `MessageActionListItems`? Or, what is intended to pass in here? Looks like it would be components as an array, right? Maybe `MessageActionListItems: Array<React.ComponentType<MessageActionListItemProps>>` or `React.ComponentType<MessageActionListItemProps>[]` could show a bit clearer what is intended to be passed in?